tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-integration-test.php

```
@@ -807,10 +922,208 @@ public function test_store_sync_result_persists_entry(): void {
+		try {
+			for ( $i = 0; $i < 4; $i++ ) {
+				$context = \WC_Stripe_Agentic_Commerce_Integration::get_delivery_context();
+
+				$expected_key = 'test' === $context['mode'] ? 'sk_test_snapshot' : 'sk_live_snapshot';
+				$this->assertSame( $expected_key, $context['secret_key'], 'The key must always belong to the mode captured in the same snapshot.' );
+			}
+		} finally {
+			remove_filter( 'option_woocommerce_stripe_settings', $alternate );
+			delete_option( 'woocommerce_stripe_settings' );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_get_delivery_context_pairs_mode_and_key_from_one_read is not named in camelCase. (undefined)

(CamelCaseMethodName)